tools/dbus_ignorelist

```diff
@@ -904,6 +904,7 @@ org.freedesktop.systemd1.Mount.Options
 org.freedesktop.systemd1.Mount.PAMName
 org.freedesktop.systemd1.Mount.PassEnvironment
 org.freedesktop.systemd1.Mount.Personality
+org.freedesktop.systemd1.Mount.PrivateBPF
```

Comment:
Why the property is added to the ignore list?
Instead, please mention the property in History section.

Reply:
I saw other similar properties like `PrivateTmp`, `ProtectProc` being added too.
What's the logic behind the ignore list?